schutzbot/mockbuild.sh

```diff
@@ -88,12 +86,14 @@ greenprint "🧬 Using mock config: ${MOCK_CONFIG}"
 greenprint "📦 SHA: ${COMMIT}"
 greenprint "📤 RPMS will be uploaded to: ${REPO_URL}"
 
-if [[ $VERSION_ID == 8.5 ]]; then
-    greenprint "📋 Updating RHEL 8 mock template for unsubscribed image"
+if [[ "$ID" == rhel ]] && ! sudo subscription-manager status; then
+    greenprint "📋 Updating RHEL 8 mock template with the latest nightly repositories"
+    # strip everything after line with # repos
     sudo sed -i '/# repos/q' /etc/mock/templates/rhel-8.tpl
     # remove the subscription check
     sudo sed -i "s/config_opts\['redhat_subscription_required'\] = True/config_opts['redhat_subscription_required'] = False/" /etc/mock/templates/rhel-8.tpl
-    cat "$RHEL85_NIGHTLY_REPO" | sudo tee -a /etc/mock/templates/rhel-8.tpl > /dev/null
+    # reuse redhat.repo
+    cat /etc/yum.repos.d/rhel8internal.repo | sudo tee -a /etc/mock/templates/rhel-8.tpl > /dev/null
```

Comment:
@ondrejbudai this fails on the rhel-9.0 runners b/c the filename is different, see https://gitlab.com/osbuild/ci/osbuild-composer/-/jobs/1608173404/. 

How do you want to handle this? Re-arrange the if conditions in this block or something else ?

Reply:
Run this block only for RHEL 8.